data/campaigns/Descent_Into_Darkness/scenarios/12_Endless_Night.cfg

```diff
@@ -826,67 +1382,127 @@
-                            message= _ "Once he chose this unlife for himself and his soldiers, he forfeited any claim to our compassion. Does it even matter who he was?"
+                            role=speakinghero
+                            message= _ "A selfish human, only greedy for power while utterly indifferent to anything but himself. A necromancer’s only purpose is to work for their own gain, nothing else."
                         [/message]
 
                         [message]
                             role=second
-                            message= _ "No, I suppose it does not."
+                            message= _ "Pure evil, that is what they are."
+                        [/message]
+
+                        [message]
+                            role=speakinghero
+                            message= _ "Indeed. We need not afford this one any compassion. Let us leave this miserable place."
                         [/message]
                     [/case]
 
                     [case]
                         value=dwarf
                         [message]
-                            speaker=Foolish Hero
-                            message= _ "Hearing the crack of his bones was satisfying."
+                            speaker=Mal Keshar
+                            message= _ "Darkness descends... and I..."
+                        [/message]
+
+                        [kill]
+                            id=Mal Keshar
+                            animate=yes
+                        [/kill]
+
+                        [message]
+                            role=speakinghero
+                            # po: regular english - the lich is nothing but a pile of bones now
+                            message= _ "Tha lich be nae but a pile o’ bones now."
                         [/message]
 
                         [message]
                             role=second
-                            message= _ "Who do you think he was before becoming the bones crushed by our hammers?"
+                            # po: regular english - I wonder what kind of human he was in life
+                            message= _ "I wonder what kind o’ human he did be in life."
                         [/message]
 
                         [message]
-                            speaker=Foolish Hero
-                            message= _ "Probably some angst-ridden mage that could not make it at that magic academy those humans have. And when he did dark magic and got outlawed from Wesnoth, he came to hide in these caves."
+                            role=speakinghero
+                            # po: regular english - I guess he was some crazy mage who wanted to make other humans his slaves. Why else do you think he would turn to dark magic?
+                            message= _ "Me guess be that he be some loonie mage who wanted ta make other humans inta ’is slaves. Why else d’ye think tha lad woul’ turn ta dark magic?"
```

Comment:
With this change, the five alternative texts for this event are:

- (human) "No more than a power-hungry mage who selfishly sought to abuse the others around him for his own gain. Necromancers are all the same. Their kind is only ever a pure evil."
- (outlaw) "Likely one of those arrogant mages who think themselves superior to everyone else. He probably learned necromancy just to subjugate people like us."
- (elf) "A selfish human, only greedy for power while utterly indifferent to anything but himself. A necromancer’s only purpose is to work for their own gain, nothing else."
- (dwarf) "Me guess be that he be some loonie mage who wanted ta make other humans inta ’is slaves. Why else d’ye think tha lad woul’ turn ta dark magic?"
- (orc) "Just some puny human. Amusing that he was more bothersome in death than in life. Annoying that it took so long to kill him."

I think it's a little repetitive that the first four all assume the lich was evil and just wanted to enslave everyone, and conversely, I liked how before this PR the dwarf's guess ("some angst-ridden mage...") was pretty much exactly right. I think it would be good if at least one of the five hypotheses were made more charitable, both to make the script less tropey and to be more compatible with lore (dwarves and elves are old species and probably know that not all necromancers are evil).

Reply:
Having one more charitable is fine, but I don't think dwarves and elves are the understanding types, despite being older species. I'll probably figure out how to adjust one of them soon.